docs/how-to/convert-react-app.md

```diff
@@ -0,0 +1,31 @@
+---
+parent: How To
+nav_order: 1
+title: Convert React App to Modular Project
+---
+
+# Convert a React App to a Modular Project
+
+To convert an existing React app project into a Modular project, start by
```

Comment:
Let's mention CRA here.

To convert an existing [Create React App](https://create-react-app.dev/) project into a Modular app, start by...